### PR TITLE
docs: Add some links

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
See https://github.com/inspektor-gadget/inspektor-gadget/issues/2151#issuecomment-2291485195

### CODE REVIEW

**Review by blanquicet — APPROVED** [review]
LGTM!

File: docs/spec/oci.md [hunk 1]
```diff
@@ -27,9 +27,7 @@ architectures are supported:
 
 Media type: `application/vnd.gadget.config.v1+yaml`
 
-The content has the type [GadgetMetadata
-struct](https://github.com/inspektor-gadget/inspektor-gadget/blob/7d12644a89217bdbf861da54cd8bd2a370754ece/pkg/gadgets/run/types/metadata.go#L136).
-It is a work in progress.
+This containers the [gadget metadata](../gadget-devel/metadata.md).
```
Comment (alban) [inline docs/spec/oci.md:30]:
```suggestion
This contains the [gadget metadata](../gadget-devel/metadata.md).
```

File: docs/spec/oci.md [hunk 2]
```diff
@@ -43,12 +41,14 @@ different media type among the following:
 ### The ebpf layer
 
 There must be exactly one layer with the ebpf media type. It must not be empty.
-Its content must be a valid ELF file.
+Its content must be a valid ELF file. See [gadget eBPF
+API](../gadget-devel/gadget-ebpf-api.md)
```
Comment (alban) [inline docs/spec/oci.md:45]:
```suggestion
API](../gadget-devel/gadget-ebpf-api.md).
```

File: docs/spec/oci.md [hunk 3]
```diff
@@ -43,12 +41,14 @@ different media type among the following:
...
 
 ### The wasm layer
 
 There must be at most one layer with the wasm media type. If present, it must
-not be empty and it must be a valid wasm file.
+not be empty and it must be a valid wasm file. See [WASM
+API](../gadget-devel/gadget-wasm-api-raw.md)
```
Comment (alban) [inline docs/spec/oci.md:51]:
```suggestion
API](../gadget-devel/gadget-wasm-api-raw.md).
```

**Review by alban — APPROVED** [review]
LGTM after a couple of dots.

**Pull request merged**